docs/modules/ROOT/pages/utils/_class_hashes.adoc

```diff
@@ -1,11 +1,11 @@
 // Version
-:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.4.4[cairo 2.4.4]
+:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.5.0[cairo 2.5.0]
 
 // Class Hashes
-:account-class-hash: 0x0402765bcede84b1267a9d4658a7737c3c41a8caf6201984c3df95577c2298a3
-:eth-account-upgradeable-class-hash: 0x03dda9bcfa854795d91d586b1a4275a68ab1ab185b33a5c00ce647c75875b0ff
-:erc20-class-hash: 0x03af5816946625d3d2c94ea451225715784762050eba736f0b0ad9186685bced
-:erc721-class-hash: 0x045c96d1b24c3dc060680e4bfd4bdc32161aefe8f8939cd4be3954c5d8688d75
+:account-class-hash: 0x07bb7a849957e721ee9e87e9844cba8ba139e87e671d4c34b17e09829b65134c
```

Comment:
After merging main the Account hashes changes from using `unwrap_syscall` instead of `unwrap`.

I wonder if we shouldn't directly use 2.5.3 in this bump. That would also affect the class hashes.

Reply:
Hmm yeah, might as well use 2.5.3. Will update